rtb/analisi_dei_requisiti_ver0.4.0.typ

```diff
@@ -0,0 +1,732 @@
+- *Trigger*: il sistema genera e invia all'utente un annuncio personalizzato relativo al punto di interesse in cui si trova.
+- *Scenario principale*: 
+  + Un utente si avvicina a un punto di interesse e il sistema genera un annuncio personalizzato che viene inviato all'utente.
+  + Nello stesso istante, nella _dashboard_, compare una notifica _pop-up_ visibile all'amministratore, con i seguenti dettagli:
+    - Utente destinatario.
+    - Punto di interesse associato.
+    - Data e ora di generazione dell'annuncio.
+- *Inclusioni*: 
+  + Scomparsa automatica della notifica di un annuncio generato (#link(<uc12>)[UC12]).
+  + Chiusura della notifica di un annuncio generato (#link(<uc13>)[UC13]).
+  + Apertura dello storico degli annunci dalla notifica (#link(<uc14>)[UC14]).
+
+// vedi uc11, l'amministratore che sarebbe attore non interagisce nemmeno
+=== UC12 - Scomparsa automatica della notifica di un annuncio generato <uc12>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato, ha accesso alla _dashboard_ del sistema e ha ricevuto una notifica di annuncio generato.
+- *Postcondizioni*: la notifica scompare automaticamente dopo un tempo prestabilito.
+- *Trigger*: l'amministratore non interagisce con la notifica.
+- *Scenario principale*: 
+  + L'amministratore riceve sulla _dashboard_ la notifica relativa ad un annuncio generato.
+  + La notifica resta visibile per un tempo predefinito, terminato il quale scomparirà automaticamente.
+
+=== UC13 - Chiusura della notifica di un annuncio generato <uc13>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato, ha accesso alla _dashboard_ del sistema e ha ricevuto una notifica di annuncio generato.
+- *Postcondizioni*: l'amministratore chiude la notifica confermando di aver preso visione.
+- *Trigger*: l'amministratore vuole chiudere la notifica di annuncio generato.
+- *Scenario principale*: 
+  + L'amministratore riceve sulla _dashboard_ la notifica relativa ad un annuncio generato.
+  + L'amministratore segnala alla notifica di aver preso visione del messaggio.
+  + La notifica viene chiusa fatta scomparire dalla _dashboard_.
+
+=== UC14 - Apertura dello storico degli annunci dalla notifica <uc14>
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato, ha accesso alla _dashboard_ del sistema e ha ricevuto una notifica di annuncio generato.
+- *Postcondizioni*: l'amministratore visualizza lo storico degli annunci.
+- *Trigger*: l'amministratore vuole visualizzare per intero l'annuncio che è appena stato generato tramite la notifica ricevuta.
+- *Scenario principale*: 
+  + L'amministratore riceve sulla _dashboard_ la notifica relativa ad un annuncio generato.
+  + L'amministratore, interagendo con la notifica, acccede allo storico degli annunci ordinati dal più recente, dove in cima sarà quindi presente l'annuncio appena generato.
+
+// vedi uc11
```

Comment:
anche qui, visualizzazione quindi esiste

Reply:
Si esatto, cambierei anche il titolo per quanto mi riguarda con "Visualizzazione"

Reply:
Attenzione poi: storico annunci o il singolo annuncio. Se devo vedermelo impresso in testa, non trovo il senso di vedere lo storico degli annunci da uno singolo generato. Poi ditemi voi la vostra